src/models/shape_loader_private.h

```diff
@@ -1,5 +1,5 @@
-#ifndef ED_MODELS_SHAPE_LOADER_H_
-#define ED_MODELS_SHAPE_LOADER_H_
+#ifndef ED_MODELS_SHAPE_LOADER_PRIVATE_H_
+#define ED_MODELS_SHAPE_LOADER_PRIVATE_H_
```

Comment:
Maybe add some comment on why you split. That might be convenient for future reference

Reply:
Done